Makefile

```diff
@@ -140,8 +140,12 @@ e2e: docker-apiserver docker-cli docker-orchestrator docker-ui docker-ui-backend
 clean-ui:
 	@(rm -rf ui/build ; echo "UI cleanup done" )
 
+.PHONY: clean-go
+clean-go:
+	@(rm -rf bin ; echo "GO executables cleanup done" )
```

Comment:
might need to be more specific here, if we `rm -rf bin` we'll delete golangci-lint and license-check etc too which I'm not sure is what we want to happen.

Reply:
Thanks for reviewing, @Tehsmash!
I think it would be good to clean every executable in the bin folder with `make clean` not only the ones we build, since we use curl to get those executables when we run `make check` etc. What do think?

Reply:
+1 I think that would be a good idea, have like clean-golangci-lint + clean-licensei